static/js/publisher/release/actions/pendingReleases.js

```diff
@@ -143,14 +145,24 @@ export function promoteChannel(channel, targetChannel) {
 }
 
 export function undoRelease(revision, channel) {
-  return {
-    type: UNDO_RELEASE,
-    payload: { revision, channel }
+  return dispatch => {
+    dispatch(
+      triggerGAEvent(
+        "click-cancel-promotion",
+        `${channel}/${revision.architectures[0]}`
+      )
+    );
+    return dispatch({
```

Comment:
does this need to be wrapped in a dispatch? or can it just return the object (I'm not actually sure without trying)?

Reply:
It looks like if I don't wrap it in a `dispatch` it doesn't change the state